### ISSUE TITLE
FF reporting weird Image Widget errors

### ISSUE THREAD

**Issue by 4SandyD** [FlutterFlow/flutterflow-issues#3570]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

Project fine (bugless) when I ended session on Monday, Aug 5. When I opened it up today (and after refreshing for update) it came up with 2 bugs related to Image widgets. When I click on first error it takes me to an onboarding page with 1 image. Deleting the image does not make the bug go away. When I click on second error it takes me to a deletion popup that doesn't even contain an image. 

### Expected Behavior

I expected the full functionality I had the previous day with no image widget errors noted.

### Steps to Reproduce

1. Click on bugs notification. 2. Click on 1st bug listed to be taken to the screen where the bug is at. 3. Delete circle image widget from page. Bug should go away. 4. Click on 2nd bug listed. 5. Note that it takes you to a popup that doesn't even contain an image widget.

### Reproducible from Blank

- [ ] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4oic/q25JNrt9K+a6IYO9WgSQUNmY9ULgkm8pBczwYF5D0PJkQRc/BQExWXL2qTWxbfE39imQe7Pfqi/zPUsIEITSsfIA6yap2bTzMUk6hMKqyO4eZfW1SAf5RC3XEyKeNnx55AOhucmQ43U2tf+vKcCnUQ+6yIFgvGs+lKdeK2SrDX1iXc2URm05KZDPz; IT4oic/q25JNrt9K+a6IYO9WgSQUNmY9ULgkm8pOGDwbfaKvPIlzaczSXFJvZ/WnY3NXKEWJhnkfwPzKj+P9Lu4oG06Zf4Raz7lpFw/hUkm/aoSuCbiwWWt9MJ5gf26s3MKrmB1ABON1WXAS1DmmH/O+SnXUHY/7Zwh9f6PHaOI=

### Visual documentation

![image](https://github.com/user-attachments/assets/7e43f082-b0a3-4b0e-bb5a-d31b279b6753)

### Environment

```markdown
- FlutterFlow version: 4.1.81
- Platform: Web
- Browser name and version: 127.0.6533.100 (Official Build) (64-bit)
- Operating system and version affected: Edition	Windows 10 Pro
Version	22H2
Installed on	‎Saturday: ‎10: ‎Oct: ‎20
OS build	19045.4651
Experience	Windows Feature Experience Pack 1000.19060.1000.0
```

### Additional Information

I'm trying to get test and run functions going again.

**Comment by msusviela** [comment]
Hi @4SandyD. So sorry you're experiencing this.<br>I'm closing this issue since it's a duplicate of this:<br>[https://github.com/FlutterFlow/flutterflow-issues/issues/3565](https://github.com/FlutterFlow/flutterflow-issues/issues/3565)<br><br>You can follow the original thread for updates. Currently our ENG team is aware of this and is deploying a fix for this that will be put as soon as possible into production.<br>Thanks for filling in.

**Comment by msusviela** [comment]
Duplicate:<br>[ENG-3186](https://linear.app/flutterflow/issue/ENG-3186/images-not-appearing-in-apk)

**Comment by 4SandyD** [comment]
I'm seeing that some are saying a fix was deployed and I was required to
update when I opened FF this morning but my issues are still present. Is
#3565 considered fixed?
Sandy D.

**Comment by msusviela** [comment]
Hi Sandy, from the ENG team they told me that you'll have to reupload the images and redeploy it. Have you try this?

**Comment by 4SandyD** [comment]
1. I do not understand what you mean by 'redeploy'. I am in development,
not production.
2. One of the errors is coming up on a popup on which I do not have an
image widget. How do I reupload an image when there isn't a widget there
for one?

Please advise ASAP.
Thanks,
S.

**Comment by msusviela** [comment]
Sandy, I'm sorry but we are unable to reproduce this issue from a blank project, can you please reach out to [support@flutterflow.io](mailto:support@flutterflow.io)? They can better assist you with ensuring your project is configured correctly

**Comment by 4SandyD** [comment]
Seriously? I did open a ticket and you closed it b/c you said it was the
same as the other. And now you are saying it isn't? They would have been
able to reproduce this if you hadn't closed my ticket, maybe?

Very disappointing.
S.

**Comment by msusviela** [comment]
So sorry for the misunderstanding. It is the same issue as the other one, which many users experienced and was confirmed and fixed. The fix for that was up and required a little workaround (in your case, since you weren't in prod you didn't need to redeployed but in some cases it was necessary). This tracker is only for issues that we're able to reproduce from blank. That issue we can no longer be replicated it because it's was fixed, but required that workaround that was mentioned also in the original thread. Our support team can ensure everything is configured correctly and they also have the right credentials to do so. Again, sorry for the inconvenience, but please contact them since they can unblock you.

**Comment by 4SandyD** [comment]
Hi Micaela:
I have contacted Support and given them all the information. It's been over
24 hours and I haven't heard from them yet. Please let me know what is
going on. I haven't been able to work on my project since Monday afternoon.
I've lost 4 whole days of development time.

It's hard to recommend your product when things like this happen.
Please help me.
Thanks,
Sandy
